students/mkdir01/lesson02/Fizz_buzz.py

```diff
@@ -0,0 +1,20 @@
+def print_fizz():
+    print("Fizz", end = "")
+
+
+def print_buzz():
+    print("Buzz", end = "")
+
+
+def FizzBuzz(x=100):
+    for num in range(x):  # count from 0 to x-1
```

Comment:
you can specify start number for range so you don't have to do `num + 1` every time.

`for num in range(1, x + 1)`